app/controllers/locations_controller.rb

```diff
@@ -1,8 +1,8 @@
 class LocationsController < ApplicationController
-  def create
-    @location = build_location
+  def update
+    @location = find_location
```

Comment:
is this ivar used in any view? I didn't see it in the profiles/show view, but I might have missed it.